featureFromKaminaga

```diff
@@ -0,0 +1,4 @@
+some code
+if(condition statement){do something}else{do other things};
```

Comment:
The format is not easy to read. I think you should refactor this line.

Reply:
OK. I got that. I will refactor it now.

Reply:
Did a good job!